src/lib/annotations/Annotator.js

```diff
@@ -236,10 +237,11 @@ class Annotator extends EventEmitter {
      * Sets the zoom scale.
      *
      * @param {number} scale - current zoom scale
-     * @return {void}
+     * @return {Annotator} Annotator instance returned for chaining
      */
     setScale(scale) {
         this.annotatedElement.setAttribute('data-scale', scale);
+        return this;
```

Comment:
When in Rome :) Good discussion around patterns here but since we don't utilize that sort of chaining in Preview, please add it in a feature change.